src/commands/ProtectionsCommands.ts

```diff
@@ -33,6 +33,38 @@ export async function execEnableProtection(roomId: string, event: any, mjolnir:
     }
 }
 
+export async function execSetProtection(roomId: string, event: any, mjolnir: Mjolnir, parts: string[]) {
+    const [protectionName, ...settingParts] = parts[2].split(".");
+    const protection = PROTECTIONS[protectionName];
+    if (protection !== undefined) {
+        const defaultSettings = protection.factory().settings
+        const settingName = settingParts[0];
+        const value = parts[3];
+
+        let newSettings: { [settingName: string]: any } = {}
+        let settingInt = 0;
+
+        let message = "did nothing";
+
+        if (defaultSettings[settingName] === undefined) {
```

Comment:
No strings?

Also, that's not going to scale up to complex settings (i.e. arrays or sub-objects) but we can cross that bridge when we reach it.